lib/model/gamers.js

```diff
@@ -1,31 +1,46 @@
-import apiLib from 'call-of-duty-api';
-import {insertIntoDatabase, queryDatabase, updateDatabaseRecords} from "../etl/utils";
-import WarzoneMapper from "../etl/mapper";
-import {GAMER_TABLE} from "../constants";
+import CallOfDutyApi from 'call-of-duty-api';
+
+import {GAMER_TABLE} from '../constants';
+
+import {insertIntoDatabase, queryDatabase, updateDatabaseRecords} from '../etl/utils';
+import WarzoneMapper from '../etl/mapper';
 import {login} from '../api_wrapper';
-const API = apiLib({platform: "battle", ratelimit: {maxRequests: 2, perMilliseconds: 1000, maxRPS: 2}});
+
+import UtilityService from './../services/UtilityService.js';
+
+const API = CallOfDutyApi({platform: 'battle', ratelimit: {maxRequests: 2, perMilliseconds: 1000, maxRPS: 2}});
```

Comment:
This shouldn't be here. All API related code should be in lib/api_wrapper.js

Reply:
In any other file besides that file. It should be:
```
import {login} from '../api_wrapper';
let API = await login();
```

Reply:
This shouldn't be here. All API related code should be in lib/api_wrapper.js
All code outside the api_wrapper should use the following to get access to the API. 

```
import {login} from '../api_wrapper';
let API = await login();

```

Reply:
Just realized this is my bad.  I didn't refactor this. I'll approve this PR since this is outside the scope of this PR.